src/components/Checkout/LoanPrice.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div class="loan-price-wrapper tw-flex">
-		<div class="loan-price-select tw-flex-grow">
+		<div v-if="price > 24" class="loan-price-select tw-flex-grow">
```

Comment:
That will work. Just need to add a `v-else` below showing the price...something like...
```
<p v-else class="tw-text-h3 tw-mb-0">
    {{ price }}
</p>